sdk/core/core-http/src/util/utils.ts

```diff
@@ -254,6 +261,7 @@ export function replaceAll(
  * @param {any} value Any entity
  * @return {boolean} - true is it is primitive type, false otherwise.
  */
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
```

Comment:
unknown should work here without any other changes, right? since we're doing a typeof check immediately

Reply:
Done.